src/components/Custom/custominput.component.tsx

```diff
@@ -14,13 +14,19 @@ const CustomInput = ({
   logo,
   ...otherProps
 }: CustomInput): JSX.Element => {
+  const [isActive, setIsActive] = useState(false);
+
   return (
     <form
-      className={`px-1 rounded-lg inline-flex ${className || ""}`}
+      className={`px-1 rounded-lg inline-flex ${className || ""} ${
+        isActive ? "border-2 border-secondary" : ""
```

Comment:
You don't have to create a state for this, you can just use tailwind's `focus:` property